logic/nodes.go

```diff
@@ -112,17 +112,20 @@ func GetNetworkNodesMemory(allNodes []models.Node, network string) []models.Node
 // UpdateNodeCheckin - updates the checkin time of a node
 func UpdateNodeCheckin(node *models.Node) error {
 	node.SetLastCheckIn()
+	//If cache enabled, only Update the Check-in TS in cache
+	if servercfg.CacheEnabled() {
+		storeNodeInCache(*node)
+		return nil
```

Comment:
updates should be stored in DB as well, here it's storing in cache and returning

Reply:
done